core/mcp.go

```diff
@@ -194,6 +194,109 @@ func (m *MCP) tools(srv *dagql.Server, typeName string) ([]LLMTool, error) {
 			// never a reason to call "sync" since we call it automatically
 			continue
 		}
+		// Hide functions from the largest and most commonly used core types,
+		// to prevent tool bloat
+		switch typeName {
+		case "Query":
+			switch field.Name {
+			case
+				"currentModule",
+				"currentTypeDefs",
+				"defaultPlatform",
+				"engine",
+				"env",
+				"error",
+				"function",
+				"generatedCode",
+				"llm",
```

Comment:
aww :P ...but probably fair

Reply:
Yeah seems safer to remove it for now :)